jupyter_book/commands/__init__.py

```diff
@@ -211,6 +212,8 @@ def page(path_page, path_output, config, execute):
         config = ""
     if not execute:
```

Comment:
a nitpick, but I find it easier to keep track if the `if` statement is for the "positive", e.g. `if execute: execute = "force"`.